frontend/src/metabase/admin/upsells/components/Upsells.styled.tsx

```diff
@@ -78,3 +77,21 @@ export const UpsellCardComponent = styled.div<Variants>`
   border: 1px solid ${upsellColors.secondary};
   background-color: var(--mb-color-bg-white);
 `;
+
+export const UpsellBannerComponent = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 1rem;
+  padding: 1rem;
+  overflow: hidden;
+  border: 1px solid var(--mb-color-border);
+  border-radius: 0.5rem;
+`;
+
+export const SecondaryCTALink = styled(ExternalLink)`
+  display: inline;
+  color: ${upsellColors.primary};
+  font-weight: 700;
+  margin-left: 0.25rem;
```

Comment:
I could've used the whitespace `{" "}` in front of the link, but it seemed to me that there is a deliberate margin between the text (copy) and the link in the designs.